File: mynlp-transform/src/main/java/com/mayabot/nlp/transform/Simplified2Traditional.java
package com.mayabot.nlp.transform;

import javax.inject.Singleton;
import java.util.TreeMap;

/**
 * 简体转繁体的词典
 *
 * @author jimichan
 */
@Singleton
public class Simplified2Traditional extends BaseTransformDictionary {

    public Simplified2Traditional() {
    }

    @Override
    public TreeMap<String, String> loadDictionary() {
        return loadFromResouce("ts/s2t.txt");
    }
}

File: modules/mynlp-transform/src/main/java/com/mayabot/nlp/transform/Simplified2Traditional.kt
package com.mayabot.nlp.transform

import java.util.*
import javax.inject.Singleton

/**
 * 简体转繁体的词典
 *
 * @author jimichan
 */
@Singleton
class Simplified2Traditional : BaseTransformDictionary() {

    override fun loadDictionary(): TreeMap<String, String> {
        return loadFromResource(RS_NAME)
    }

    companion object {
        private val RS_NAME = "ts-dict/s2t.txt"
    }
}
